rotkehlchen/api/v1/schemas.py

```diff
@@ -675,6 +676,19 @@ def _validate_historical_price_oracles(
         )
 
 
+class ExchangeLocationIDSchema(Schema):
+    name = fields.Str(required=True)
```

Comment:
As stated in the docs `fields.Str` is an alias for `fields.String` but we always use `fields.String`